shell/platform/fuchsia/flutter/integration_flutter_tests/embedder/BUILD.gn

```diff
@@ -0,0 +1,74 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+assert(is_fuchsia)
+
+import("//build/fuchsia/sdk.gni")
+import("//flutter/tools/fuchsia/fuchsia_archive.gni")
+
+group("tests") {
+  testonly = true
+  deps = [
+    ":flutter-embedder-test",
+    "//flutter/shell/platform/fuchsia/flutter/integration_flutter_tests/embedder/child-view",
+    "//flutter/shell/platform/fuchsia/flutter/integration_flutter_tests/embedder/parent-view",
+  ]
+}
+
+executable("flutter-embedder-test-bin") {
+  testonly = true
+
+  output_name = "flutter-embedder-test"
+
+  sources = [
+    "flutter-embedder-test.cc",
+    "flutter-embedder-test.h",
+  ]
+
+  # This is needed for //third_party/googletest for linking zircon symbols.
+  libs = [ "$fuchsia_sdk_path/arch/$target_cpu/sysroot/lib/libzircon.so" ]
+
+  include_dirs = [ "//flutter/shell/platform/fuchsia/flutter/integration_flutter_tests/fuchsia_testing" ]
+
+  public_deps = [ "//third_party/googletest:gtest" ]
+
+  # The use of these dependencies is temporary and will be moved behind the
+  # embedder API.
+  flutter_deps = [
```

Comment:
Shouldn't need these; these are internal runner dependencies and the test is using a prebuilt runner

Reply:
done